### ISSUE TITLE
Check if non-bash shell is executing script?

### ISSUE THREAD

**Issue by gdevenyi** [kvz/bash3boilerplate#100]
We use lots of bashisms, for good reason.

Perhaps we should check and bail out if a non-bash shell is running the script?

One way to check: https://askubuntu.com/questions/97522/how-can-my-script-determine-whether-its-being-run-by-bash-or-dash

**Comment by kvz** [comment]
Yes I like it! And if people want to force it, they can just run with `BASH_VERSION=force` or similar
